### PR TITLE
[RAINCATCH-728] Remove appforms related documentation

### PR DESCRIPTION
**Pull request by JameelB**
# What
Remove appforms related configuration and documentation

# JIRA
https://issues.jboss.org/browse/RAINCATCH-728

### CODE REVIEW

**Review by witmicko — APPROVED** [review]
LGTM

File: getting-started.adoc [hunk 1]
```diff
@@ -47,26 +47,6 @@ RHMAP provides a Raincatcher project template to help you run the demonstration.
 
 .. Click *Finish* at the bottom of the page when the project is created. 
 
-. Create a blank forms theme as follows:
```
Comment (wtrocki) [inline getting-started.adoc:50]: After we publish this change we will need to send email to feedhenry-raincatcher to inform users that we removed forms and documentation was updated.

File: running-locally.adoc [hunk 2]
```diff
@@ -7,11 +7,7 @@
 [[rhmap]]
 === RHMAP
 
-The demo solution currently requires an instance of RHMAP. It can be a local or remote instance of RHMAP, but must be configured as outlined in the Raincatcher link:getting-started.adoc[Getting Started Guide] so that the local *raincatcher-demo-cloud* app can access the following RHMAP features:
-
-* Forms
-* Themes
-* Drag & Drop App Configuration
+The demo solution currently requires an instance of RHMAP. It can be a local or remote instance of RHMAP, but must be configured as outlined in the Raincatcher link:getting-started.adoc[Getting Started Guide]
```
Comment (wtrocki) [inline running-locally.adoc]: As far I understand it after we remove forms there will be no more ties to RHMAP.

**Review by wtrocki — CHANGES_REQUESTED** [review]
Local dev setup guide still has references to forms.

File: running-locally.adoc (hunk 2 above)
Comment (JameelB) [reply running-locally.adoc]: I asked @nialldonnellyfh about this and he said we do still need RHMAP for $fh.auth

**Review by wtrocki — APPROVED** [review]

File: running-locally.adoc (hunk 2 above)
Comment (wtrocki) [reply running-locally.adoc]: Ok. So let's incorporate this into this statement.

**Pull request merged**